core/src/test/resources/HistoryServerExpectations/excludeOnFailure_node_for_stage_expectation.json

```diff
@@ -81,7 +93,19 @@
           "remoteBytesRead" : 0,
           "remoteBytesReadToDisk" : 0,
           "localBytesRead" : 0,
-          "recordsRead" : 0
+          "recordsRead" : 0,
+          "pushBased": {
```

Comment:
This should be "Push Based shuffle" right ?

Reply:
This is the API response that serialized based on the [case classes in api.scala here](https://github.com/apache/spark/blob/7259a8c0147d9da70848ef2e7499a1712a4c4741/core/src/main/scala/org/apache/spark/status/api/v1/api.scala#L384). We only write "Push Based Shuffle" when logging to the event file

Reply:
Do you recommend changing this to "pushBasedShuffle" ? to note:  it is inside the context of shuffle read metrics)

Reply:
Yes, that would be better.

Reply:
Changed it to "pushBasedShuffle", thanks for the suggestion.